src/components/nve-textarea/nve-textarea.component.ts

```diff
@@ -92,6 +92,9 @@ export default class NveTextarea extends LitElement {
    */
   @property() inputmode?: 'none' | 'text' | 'decimal' | 'numeric' | 'tel' | 'search' | 'email' | 'url';
 
+  /** Antall rader i textarea-taggen */
```

Comment:
For de som ikke kjenner <textarea>, kunne du skrevet litt mindre "teknisk" beskrivelse. Se gjerne på hva [Mozilla skriver](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/textarea#rows)